config.yaml

```diff
@@ -37,7 +37,9 @@ params:
   mash:
     sketch_size: 1000
   kraken:
-    kraken_db: test_data/kraken_test_db
+    # kraken_db: test_data/kraken_test_db
+    kraken_db: /home/qiz173/Works/WoL/profiling/ref107/kraken
```

Comment:
do we have a general solution for those directories. Does not feel correct to use files in personal home dirs.

Reply:
Yes we do. Oh sorry. Actually this should be `kraken_db: test_data/kraken_test_db`. I was testing using some of my own databases. Let me change.